README.md

```diff
@@ -1,16 +1,23 @@
-# Faded Text
+<p align="center">
+<img src="./docs/images/cover.png" height="220" />
+</p>
 
-[![Build Status](https://shields.io/github/actions/workflow/status/surfstudio/surf-flutter-faded-text/main.yml?logo=github&logoColor=white)](https://github.com/surfstudio/surf-flutter-faded-text)
-[![Pub Version](https://img.shields.io/pub/v/faded_text?logo=dart&logoColor=white)](https://pub.dev/packages/faded_text)
-[![Pub Likes](https://badgen.net/pub/likes/faded_text)](https://pub.dev/packages/faded_text)
-[![Pub popularity](https://badgen.net/pub/popularity/faded_text)](https://pub.dev/packages/faded_text/score)
-![Flutter Platform](https://badgen.net/pub/flutter-platform/faded_text)
+## Overview 🐄 
 
-## Description
+By default, text fade effects occur in the bottom-up direction if the content contains multiple lines. But sometimes it is necessary to make the fade effect from left to right, as it is done for single-line text in Flutter.
+This package solves this problem. It allows you to insert fade when wrap text on overflow from left to right.
```

Comment:
More concise: 

"Enhance your Flutter app with this package, designed to seamlessly integrate left-to-right fade effects for multi-line text. Perfect for when you need that single-line text elegance in more complex layouts."

Reply:
Resolve in #3